docs/ScalarDLSupportedDatabase.md

```diff
@@ -0,0 +1,48 @@
+# Set up a Scalar DL supported database
+
+This is a database setup document for the Scalar DL manual deployment. Scalar DL can run on various databases. 
+You can choose any one of the supported databases based on your convenience.
+
+## DynamoDB
+
+By default, Amazon DynamoDB is available for all AWS users, so you do not need to configure anything manually.
+Scalardl schema tool will help you to configure DynamoDB.
```

Comment:
Should it be a job? (since you haven't really described what the schmea tool is and users don't really notice it if they follow the other manual guide )

Reply:
> If you use the helm chart, you don't really notice if you are using the schema tool, right?

Yes. Specified as `Scalardl schema tool helm charts`